src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -60,33 +59,28 @@ interface Props {
   colony: Colony;
   currentDomainId?: number;
   maxAvatars?: number;
-  // members?: ColonyContributor[] | ColonyWatcher[];
+  members?: Contributor[] | Watcher[];
```

Comment:
Does it still make sense for it to be optional? You seem to provide a fallback of empty array in `ColonyMembersWidget`